common/static/common/js/utils/edx.utils.validate.js

```diff
@@ -159,9 +160,15 @@
                                         content = _.sprintf(_fn.validate.msg[key], context);
                                     }
 
-                                    txt.push(_fn.validate.template({
-                                        content: content
-                                    }));
+                                    if (['username', 'email'].indexOf($el.attr('name')) > -1) {
+                                        txt.push(_fn.validate.suppressTextTemplate({
+                                            content: content
+                                        }));
+                                    } else {
+                                        txt.push(_fn.validate.template({
+                                            content: content
+                                        }));
+                                    }
```

Comment:
And from here pass it like this:
```suggestion
                                    suppressAttr = '';
                                    if (['username', 'email'].indexOf($el.attr('name')) > -1) {
                                        suppressAttr = 'data-hj-suppress';
                                    }
                                    txt.push(_fn.validate.template({
                                        content: content,
                                        suppressAttr: suppressAttr
                                    }));
```